core/package.json

```diff
@@ -8,11 +8,13 @@
     "@babel/node": "^7.2.2",
     "@babel/preset-env": "^7.4.2",
     "@babel/register": "^7.0.0",
+    "acorn": "^6.1.1",
```

Comment:
What is it for?

Reply:
This is solving a warning when doing `npm install`